integration-tests/tests/run-workflow.js

```diff
@@ -33,7 +33,7 @@ const testRunWorkflowFn = _.flow(
 
   await waitForNoSpinners(page)
   await click(page, clickable({ text: 'Select Data' }))
-  await click(page, input({ labelContains: 'Choose specific rows to process' }))
+  await click(page, input({ labelContains: 'Choose specific test_entitys to process' }))
```

Comment:
I think I see where you were going with this, but it scanned kind of weirdly as I was reading it:

```suggestion
  await click(page, input({ labelContains: 'Choose specific test_entities to process' }))
```

Reply:
Unfortunately, since we pluralize by adding an 's', this is in fact what shows up when the entity name is 'test_entity'. There are more sophisticated pluralization strategies out there, but it's not clear that they would always be better. E.g., should 'sample_from_goose' pluralize to 'sample_from_geese'? We could also say 'Choose specific test_entity rows to process', but that might be a bit clunky.